### DISCUSSION TITLE
Demo install.sh // Istio CRDs missing

### DISCUSSION THREAD

**Discussion by heikoliedtke** [eclipse-lmos/lmos-demo#21] [General]
Trying to setup the demo and follow the instructions. Struggle with:

istioctl install -y
kubectl apply -f istio/istio-operator.yaml

yaml is referring to CRD "install.istio.io/v1alpha1" which is not available after following the setup.
Tried with VSCode DevContainer minikube and also with GCP CloudShell minikube...
Any thoughts?

**Comment by RobWin** [comment]
Hi thank you. I can check when I'm back from vacation.
We are using Minikube in devcontainer. Unfortunately the Minikube and Istio version is not yet fixed in the devcontainer.json file. Maybe there has been a new version.

**Comment by kaikreuzer** [comment]
Hi @heikoliedtke, I can confirm the issue - we will have a look at it asap!

**Comment by kaikreuzer** [comment]
I've created https://github.com/lmos-ai/lmos-demo/issues/22 for it.

**Comment by kaikreuzer** [comment]
@heikoliedtke [This change](https://github.com/lmos-ai/lmos-demo/pull/23/files) fixes it for me.
FWIW: Please note that we are currently transitioning the project to the Eclipse Foundation and therefore the current `main` branch is not yet fully functional - you might want to use the `main-archive` branch for the time being.

**Comment by heikoliedtke** [comment]
Thanks for the swift response and fix @kaikreuzer. Will continue with this as soon as I am back in office :-)

**Comment by jozzy** [comment]
Looking forward to your valuable feedback Heiko. And really appreciate you
taking time to check out LMOS.

Best Regards,
Arun

**Comment by kaikreuzer** [comment]
@heikoliedtke Just as a heads-up: It took a moment, but the lmos-demo should now be fully functional again. Please have a try and let us know in case you come across any hurdles!

**Comment by heikoliedtke** [comment]
Thanks @kaikreuzer , appreciate the information. Will continue my setup, also want to try a  GKE deployment soon...
